src/components/stack/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+const Stack = ({
+	additionalClassNames,
+	alignment,
+	children,
+	direction,
+	divider,
+	flexType,
+	justification,
+	gap,
+	wrap,
+}) => {
+	const defaultAndAdditionalClassnames = `c-stack${additionalClassNames ? ` ${additionalClassNames}` : ''}`;
```

Comment:
This looks really confusing at first glance.

```suggestion
	const defaultAndAdditionalClassnames = additionalClassNames ? `c-stack ${additionalClassNames}` : 'c-stack';
```
Additionally, I don't know how I feel about `additionalClassNames` being the property value.  I think it should mostly be transparent to the user that they want `className`s applied to something.  If we want to append to that list, we should do it transparently.  So I'd suggest one step further in removing `additionalClassNames` as a property and just look at the `className` value passed in.
```
const Stack = ({ className }) => (<div className={ className ? `c-stack ${className}` : 'c-stack' } />);
```

Reply:
Yeah I was following convention from other components that have been built, but I agree it's a little error prone for the use maybe

Reply:
The generator has this implementation that all new components should be based from - https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-1.20/_templates/component/new/default-entry.ejs.t#L9

Reply:
@matthewroach 

is it too late to change this?  `additionalClassNames` just seems like overkill where `className` would work just fine here and be less of a learning curve since `className` is React common convention.

Reply:
I think we can change the prop name to be `className` I much prefer that

Reply:
I'll put this convo on slack, if we need to we can make  story to update it.

Reply:
https://github.com/WPMedia/arc-themes-components/pull/26

Reply:
@nschubach Added your refactoring suggestion